lib/client.js

```diff
@@ -1114,9 +1124,17 @@ const PreClient = crypto => {
           }
         }
       )
+
       const patchResponse = await checkStatus(patchRequest)
       const patchJson = await patchResponse.json()
-      // Delete the temporary file, here encryptedFile.
+
+      fs.unlink(encryptedFile, err => {
```

Comment:
I think this will likely error when it's used in the browser sdk.   The fs load might also be an issue.  We can take a look before this gets shipped.

Reply:
I added some if statements to check whether the code is running in node or browser and only use the file system if it's running in node.